### PR TITLE
Completing docs for all public org API endpoints

### PR DESCRIPTION
**Pull request by bgrenon**
- Added remaining endpoints
- Split out consumption operations to separate page

Preview:
- https://neon-next-git-bgrenon-orgs-api-refresh-neondatabase.vercel.app/docs/manage/orgs-api
- https://neon-next-git-bgrenon-orgs-api-refresh-neondatabase.vercel.app/docs/manage/orgs-api-consumption

### CODE REVIEW

File: content/docs/manage/api-keys.md [hunk 1]
```diff
@@ -13,9 +13,22 @@ Most actions performed in the Neon Console can also be performed using the [Neon
...
-- **Project-scoped organization API key** — These keys are scoped to a specific project within an organization. They provide [member-level access](/docs/manage/organizations#user-roles-and-permissions) to the specified project, and only that project. They cannot perform organization-related actions or destructive project operations like project deletion.
+- **Personal API key** — These keys are tied to your individual Neon account. They can:
+
+  - Access all your personal projects
+  - Access organization projects when including an `org_id` parameter and you are a member of that organization
+
+- **Organization API key** — These keys are scoped to a specific organization. They:
+
+  - Full [admin-level access](/docs/manage/organizations#user-roles-and-permissions) to all projects within that organization
+  - Do not required an `org_id` parameter
```
Comment (danieltprice) [inline content/docs/manage/api-keys.md:24]:
```suggestion
  - Do not require an `org_id` parameter
```

File: content/docs/manage/orgs-api.md [hunk 2]
```diff
@@ -31,34 +33,106 @@ curl --request GET \
...
 
 ![finding your organization ID from the settings page](/docs/manage/orgs_id.png)
 
-## Creating API keys
+## Create API keys
+
+There are two types of organization API keys:
+
+- **Organization API keys** — These keys provide admin-level access to all organization resources, including projects, members, and settings
+- **Project-scoped organization API keys** — These keys provide limited,member-level access to specific projects within the organization
```
Comment (danieltprice) [inline content/docs/manage/orgs-api.md:64]:
```suggestion
- **Project-scoped organization API keys** — These keys provide limited, member-level access to specific projects within the organization
```

**Review by danieltprice — APPROVED** [review]

**Review by kszafran — COMMENTED** [review]
👌🏻

**Review by kszafran — APPROVED** [review]

File: content/docs/manage/orgs-api.md [hunk 3]
```diff
@@ -1,10 +1,12 @@
...
 enableTableOfContents: true
 updatedOn: '2024-12-17T13:59:40.784Z'
 ---
 
-Learn how to manage Neon Organizations using the Neon API, including creating and using organization-specific API keys, creating projects, transferring projects, and retrieving consumption metrics.
+Learn how to manage Neon Organizations using the Neon API, including managing organization API keys, working with organization members, and handling member invitations.
+
+## Personal vs organization API keys
 
 You can authenticate your API requests using either of these methods:
```
Comment (bgrenon) [inline content/docs/manage/orgs-api.md:11]:
```suggestion
You can authorize your API requests using either of these methods:
```

**Pull request merged**